src/sql/workbench/services/objectExplorer/browser/asyncServerTreeDataSource.ts

```diff
@@ -51,18 +52,16 @@ export class AsyncServerTreeDataSource implements IAsyncDataSource<ConnectionPro
 			} else if (element instanceof ConnectionProfileGroup) {
 				return element.getChildren();
 			} else if (element instanceof TreeNode) {
-				if (element.children) {
-					return element.children;
-				} else {
-					return await this._objectExplorerService.resolveTreeNodeChildren(element.getSession()!, element);
-				}
+				return await this._objectExplorerService.resolveTreeNodeChildren(element.getSession()!, element);
 			}
 		} catch (err) {
+			const errorMessage = getErrorMessage(err) ?? err;
```

Comment:
This will always return a string so the null coalesce part is unnecessary.

```suggestion
			const errorMessage = getErrorMessage(err);
```